### PR TITLE
PLT-1831 Add statuses to centre channel profile pictures

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Add statuses to the centre channel and also create a re-usable ProfilePicture component. Thanks to @asaadmahmood for the UI work.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-1831

#### Checklist
- [x] Has UI changes

### CODE REVIEW

File: webapp/components/post_view/components/post.jsx [hunk 1]
```diff
@@ -105,11 +106,11 @@ export default class Post extends React.Component {
             return true;
         }
 
-        if (!Utils.areObjectsEqual(nextProps.user, this.props.user)) {
+        if (nextProps.status !== this.props.status) {
             return true;
         }
 
-        if (nextProps.emojis !== this.props.emojis) {
```
Comment (enahum) [inline webapp/components/post_view/components/post.jsx]: why is the check of the emojis no longer needed?

Comment (jwilander) [inline webapp/components/post_view/components/post.jsx]: It was there twice by accident so I removed it once

**Comment by enahum** [comment]
+1

**Comment by coreyhulen** [comment]
+1, needs a rebase

**Pull request merged**